### ISSUE TITLE
Is there a way to preserve the "rotation" data of original file?

### ISSUE THREAD

**Issue by awerhane** [testfairy-blog/VideoCompressionTutorial#6]
Hi,

Thank you so much fo creating this project. I'm not sure if I'm missing something, but is there a way to preserve all the exif data of the original file rather than transforming the video?

For example, I have a video taken from the Front Camera in portrait mode. When I export the file directly and look at the output from exiftool, I see:
rotation 90.

But if I use the .fixForFrontCamera CompressionTransform, the compressed file has (but it does render in correct orientation):
rotation 0.

If I use .keepSame, the rotation information isn't preserved (and the video appears rotated 90˚ counter clockwise).

TLDR; Is there a configuration to use this compression tool to compress the file, but preserve as much meta data as possible (specifically the "rotation)?

I'd be happy to provide links to files if that helps clarify, but I'm hoping I'm just missing an obvious option.

Thanks!

-Andrew

**Comment by diegoperini** [comment]
Hi Andrew,

I believe what you are describing is a missing feature that deserves its own enum entry like `.keepSamePreservingExif`. It was beyond the scope of this tutorial when I've first written it but let me do research on it for a little before all hope is lost. I'll let you know if I have a solution.

Thanks for the kind words.

**Comment by akaashdev** [comment]
Hi @awerhane 

Refer this [PR](https://github.com/testfairy-blog/VideoCompressionTutorial/pull/5) or try using this [fork](https://github.com/akaashdev/SwiftVideoCompressor). Have fixed this issue there. 
Apologies for the late reply.

-Akaash Dev

**Comment by diegoperini** [comment]
I'm closing this since there already exists a fork with the needed fix. Feel free to reopen if that doesn't solve the issue.
